mftf/2.3/test/action-groups.md

```diff
@@ -18,8 +18,8 @@ The following diagram shows the structure of an MFTF action group:
 
 The following conventions apply to MFTF action groups:
 
--  All action groups are declared in XML files and stored in the `<module>/ActionGroup/` directory.
--  Every file name ends with `ActionGroup`, such as `LoginToAdminActionGroup.xml`.
+- All action groups are declared in XML files and stored in the `<module>/ActionGroup/` directory.
+- Every file name ends with `ActionGroup`, such as `LoginToAdminActionGroup.xml`.
```

Comment:
ends with `ActionGroup.xml`?

Reply:
will change to `LoginToAdminActionGroup`